core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -723,23 +1319,29 @@ class EntityApiServiceSpec extends ApiServiceSpec {
       }
   }
 
-  /*
-    following 2 tests disabled until entity deletion is re-implemented (see GAWB-422)
-   */
-  ignore should "*DISABLED* return 204 on entity delete" in withTestDataApiServices { services =>
```

Comment:
These basic deleted tests have been moved upward in this file.  These changes are new tests.